lhfit doesn't support single gain telescopes

Discovered because in the new test file, tel_id=2 is MAGIC with a single gain, not the LST-2:

```
        config['source_config']['EventSource']['allowed_tels'] = [1, 2]
        config['lh_fit_config']["no_asymmetry"] = True
        config['lh_fit_config']["use_weight"] = True
        config['lh_fit_config']["verbose"] = 0
>       r0_to_dl1(mc_gamma_testfile, custom_config=config, output_filename=tmp_path / "tmp.h5")

lstchain/tests/test_lstchain.py:128: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
lstchain/reco/r0_to_dl1.py:445: in r0_to_dl1
    lhfit_fitter = TimeWaveformFitter(subarray=subarray, config=Config(lhfit_fitter_config))
lstchain/reco/reconstructor.py:82: in __init__
    self.template_dict[tel_id] = NormalizedPulseTemplate.load_from_eventsource(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'lstchain.data.normalised_pulse_template.NormalizedPulseTemplate'>
eventsource_camera_readout = CameraReadout(name='MAGICCam', sampling_rate=1.639999960899354 GHz, n_channels=1, n_pixels=1039, n_samples=50, n_samples_long=None)
kwargs = {}, t = array([0.00000000e+00, 1.52439028e-02, 3.04878056e-02, ...,
       2.43445128e+01, 2.43597567e+01, 2.43750006e+01])

    @classmethod
    def load_from_eventsource(cls, eventsource_camera_readout, **kwargs):
        """
        Load a pulse template from an event source camera readout.
        Read the sampling rate to create a time variable reaching
        9 ns at the HG maximum
    
        Parameters
        ----------
        cls: This class
        eventsource_camera_readout: `CameraReadout`
            CameraReadout object obtained from the LST event source
    
        Return
        ----------
        cls(): Instance of NormalizedPulseTemplate receiving the information
               from the input file
    
        """
    
        t = eventsource_camera_readout.reference_pulse_sample_time.to_value('ns')
>       hg, lg = eventsource_camera_readout.reference_pulse_shape
E       ValueError: not enough values to unpack (expected 2, got 1)

lstchain/data/normalised_pulse_template.py:193: ValueError
```

--- a/lstchain/data/normalised_pulse_template.py
+++ b/lstchain/data/normalised_pulse_template.py
@@ -190,7 +190,11 @@ class NormalizedPulseTemplate:
         """
 
         t = eventsource_camera_readout.reference_pulse_sample_time.to_value('ns')
-        hg, lg = eventsource_camera_readout.reference_pulse_shape
+        pulses = eventsource_camera_readout.reference_pulse_shape
+        if len(pulses) == 2:
+            hg, lg = pulses[0], pulses[1]
+        if len(pulses) == 1:
+            hg, lg = pulses[0], pulses[0]
         i = np.argmax(hg)
         t = t - t[i] + 9.0
         return cls(amplitude_HG=hg, amplitude_LG=lg, time=t, **kwargs)

